swagger.yaml

```diff
@@ -1020,7 +1019,7 @@ paths:
       description: |
         Deletes a reply
 
-        Method allows system administrators and metaform administrators to delete a reply. Otherwise user must have reply:edit scope permission for given reply. Permissions are handled by the Keycloak authorization system.
+        Method allows system administrators, metaform administrators and metaform managers to delete a reply. Otherwise user must have reply:edit scope permission for given reply. Permissions are handled by the Keycloak authorization system.
```

Comment:
Are you sure that managers should be allowed to delete replies?

Reply:
That's copypaste error, removed managers from it.